frontend/routes/components/panel/panel.component.tsx

```diff
@@ -24,16 +24,19 @@ import { MuiTheme, theme } from '../../../styles';
 import { Container, Title, Content } from './panel.styles';
 
 interface IProps {
-	title: string;
+	title?: string;
 	children: any;
 	paperProps?: any;
+	renderCustomTitle?: any;
```

Comment:
I would have gone with only "title" without the renderCustomTitle that could be a string or a JSX.Element . This is only to take in consideration, not needed for the pr.

Reply:
Good point! ;)